### ISSUE TITLE
Updating documentation with new functionality

### ISSUE THREAD

**Issue by arstrube** [cpputest/cpputest.github.io#28]
@basvodde - I am currently working on a PR to document the new functionality that will be part of CppUTest v3.8 (and also update some that has always been missing)

1. You can view the updated pages at http://arstrube.github.io.
2. Would it be possible for you to link cpputest with Coveralls? It doesn't really make sense to display coverage from my fork, which is the only thing I can do right now. I really think the badge should be here.
3. The Doxygen section is not really useful at all, in my opinion. Improving it would involve a lot of work (and probably some comments in the code) so my suggestion is to remove it:
    - It does not include CppUTestExt
    - It displays all the internal stuff nobody is interested in
    - It does not give a good summary of what would be truly important for a user (e.g. Testharness.h, Testharness_c.h, CommandlineTestRunner.h, UtestMacro.h MockSupport.h, Mocksupport_c.h, MockExpectedCall.h and MockActualCall.h and probably a few others), especially as the links to source file are 404.

**Comment by arstrube** [comment]
@basvodde No. 2 is still open. I don't think it's a good idea if the badge points to my fork, but it seems that cpputest/cpputest ins not properly hooked up to Appveyor, and the badge stays grey.

Could you perhaps make sure the proper connection is made in Appveyor? Then let me know and I'll change the badge.

**Comment by basvodde** [comment]
Uhm

**Comment by basvodde** [comment]
How to do that?

**Comment by arstrube** [comment]
Go to https://coveralls.io/. 
Click "Sign up for free".
Click "Github signup".
As of this point, I can't help you because it knows me and I don't get the page you would. Essentially, you sign up with the cpputest Github account, and you add the repository to Coveralls.
You have to authorize it at the Github end, but that page will come up automatically.
That's it.
